build.gradle.kts

```diff
@@ -8,28 +8,34 @@ plugins {
 }
 
 java {
-	sourceCompatibility = JavaVersion.VERSION_16
-	targetCompatibility = sourceCompatibility
+	toolchain.languageVersion.set(JavaLanguageVersion.of(16))
 }
 
 repositories {
 	mavenCentral()
 	maven { url = uri("https://papermc.io/repo/repository/maven-public/") }
-	maven { url = uri("https://mvn.intellectualsites.com/content/repositories/releases/") }
-	maven { url = uri("https://mvn.intellectualsites.com/content/repositories/thirdparty/") }
+	maven { url = uri("https://mvn.intellectualsites.com/content/groups/public/") }
 }
 
 dependencies {
-	compileOnlyApi("io.papermc.paper:paper-api:1.17-R0.1-SNAPSHOT")
+	compileOnlyApi("io.papermc.paper:paper-api:1.17.1-R0.1-SNAPSHOT")
 	compileOnly("org.jetbrains:annotations:20.1.0")
-	compileOnlyApi("com.fastasyncworldedit:FAWE-Bukkit:1.17-23")
+	compileOnlyApi("com.fastasyncworldedit:FAWE-Bukkit:1.17-44")
 	implementation("org.incendo.serverlib:ServerLib:2.2.0")
 	implementation("org.bstats:bstats-bukkit:2.2.1")
 	implementation("org.bstats:bstats-base:2.2.1")
 }
 
+tasks.compileJava.configure {
+	options.release.set(11)
+}
+
+configurations.all {
+	attributes.attribute(TargetJvmVersion.TARGET_JVM_VERSION_ATTRIBUTE, 16)
+}
```

Comment:
Unneeded like outlined in my previous comment. We don't need to enforce Java 16 for no good reason.

Reply:
Humm, without this setting, build fails:

```java
Execution failed for task ':compileJava'.
> Could not resolve all files for configuration ':compileClasspath'.
   > Could not resolve io.papermc.paper:paper-api:1.17.1-R0.1-SNAPSHOT.
     Required by:
         project :
      > No matching variant of io.papermc.paper:paper-api:1.17.1-R0.1-SNAPSHOT:20210707.082846-3 was found. The consumer was configured to find an API of a library compatible with Java 11, preferably in the form of class files, preferably optimized for standard JVMs, and its dependencies declared externally but:
          - Variant 'apiElements' capability io.papermc.paper:paper-api:1.17.1-R0.1-SNAPSHOT declares an API of a library, packaged as a jar, and its dependencies declared externally:
              - Incompatible because this component declares a component compatible with Java 16 and the consumer needed a component compatible with Java 11
              - Other compatible attribute:
                  - Doesn't say anything about its target Java environment (preferred optimized for standard JVMs)
          - Variant 'javadocElements' capability io.papermc.paper:paper-api:1.17.1-R0.1-SNAPSHOT declares a runtime of a component, and its dependencies declared externally:
              - Incompatible because this component declares documentation and the consumer needed a library
              - Other compatible attributes:
                  - Doesn't say anything about its target Java environment (preferred optimized for standard JVMs)
                  - Doesn't say anything about its target Java version (required compatibility with Java 11)
                  - Doesn't say anything about its elements (required them preferably in the form of class files)
          - Variant 'runtimeElements' capability io.papermc.paper:paper-api:1.17.1-R0.1-SNAPSHOT declares a runtime of a library, packaged as a jar, and its dependencies declared externally:
              - Incompatible because this component declares a component compatible with Java 16 and the consumer needed a component compatible with Java 11
              - Other compatible attribute:
                  - Doesn't say anything about its target Java environment (preferred optimized for standard JVMs)
          - Variant 'sourcesElements' capability io.papermc.paper:paper-api:1.17.1-R0.1-SNAPSHOT declares a runtime of a component, and its dependencies declared externally:
              - Incompatible because this component declares documentation and the consumer needed a library
              - Other compatible attributes:
                  - Doesn't say anything about its target Java environment (preferred optimized for standard JVMs)
                  - Doesn't say anything about its target Java version (required compatibility with Java 11)
                  - Doesn't say anything about its elements (required them preferably in the form of class files)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.


```

Reply:
That'll need to stay then.